<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientManager</class>
 <widget class="QMainWindow" name="ClientManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Manager</string>
  </property>
  <property name="styleSheet">
   <string>
    QMainWindow {
      background-color: white;
    }
    QLineEdit, QTextEdit {
      padding: 8px;
      border: 1px solid #ddd;
      border-radius: 4px;
      background-color: white;
    }
    QPushButton#addButton, QPushButton#editButton, QPushButton#deleteButton {
      background-color: #4c4cfa;
      color: white;
      border: none;
      border-radius: 4px;
      padding: 10px;
      font-weight: bold;
    }
    QPushButton#deleteButton {
      background-color: #fa4c4c;
    }
    QTableWidget {
      border: 1px solid #ddd;
      border-radius: 4px;
      background-color: white;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Client Manager</string>
      </property>
      <property name="styleSheet">
       <string>
        font-size: 24px;
        font-weight: bold;
        margin-bottom: 20px;
       </string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchEdit">
        <property name="placeholderText">
         <string>Search by client name, project, location...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="clientsTable">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <column>
       <property name="text">
        <string>Client Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Project Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Location</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Contact Phone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>ADD CLIENT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>EDIT CLIENT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>DELETE CLIENT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>